package.json

```diff
@@ -73,5 +76,7 @@
     "hystrix",
     "rate-limiting"
   ],
-  "dependencies": {}
+  "dependencies": {
+    "core-js": "^3.3.3"
```

Comment:
This should be a `devDependency` shouldn't it?

Reply:
Nope! More info:
https://babeljs.io/blog/2019/03/19/7.4.0

The core-js package is a light suite of polyfills for all the major new JS features. With babel `useBuiltIns`, it will dynamically add the imports for any core-js polyfills to the dist folder. These polyfills will bloat the distribution a  tiny bit, but this package will be able to serve a much wider audience as a result. If you prefer to leave the polyfills out, you can remove coreJS and just use babel preset-env, with no args, to transpile only the syntactic sugar, without adding any polyfills. Given that this package is marketed for browsers, I think the overhead is worth it.

Note there's also a second `regenerator-runtime` polyfill dependency that Babel can generate imports for. I didn't include it as a dependency in this PR because you don't use generators or async/await code. If you would prefer maintaining async/await code, then you can easily add the `regenerator-runtime` dependency and a single `require("regenerator-runtime/runtime");` in your entrypoint (like described in the babel blog). This will transpile your bleeding edge ES2019 into browser-native ES5 at build time.